strawberry/types/lazy_type.py

```diff
@@ -59,6 +58,9 @@ def __class_getitem__(cls, params: Tuple[str, str]) -> "Self":
 
         return cls(type_name, module, package)
 
+    def __or__(self, other: Other) -> Union[Self, Other]:
+        return Union[self, other]
```

Comment:
Learned something new today from these two lines. I was wondering how they behave at runtime, but looks like they just do what we expect them to do.